tests/e2e/util.go

```diff
@@ -1896,6 +1962,7 @@ func bringSvcK8sAPIServerUp(ctx context.Context, client clientset.Interface, pvc
 
 //pvcHealthAnnotationWatcher polls the health status of pvc and returns error if any
 func pvcHealthAnnotationWatcher(ctx context.Context, client clientset.Interface, pvclaim *v1.PersistentVolumeClaim, healthStatus string) error {
+	framework.Logf("Looking Health Annotation for pvclaim %v", pvclaim.Name)
```

Comment:
"Looking Health Annotation for pvclaim %v" -> "Waiting for health annotation for  pvclaim %v to be %v"

Reply:
done